src/supautils.c

```diff
@@ -239,11 +239,8 @@ supautils_hook(PROCESS_UTILITY_PARAMS)
 	{
 		/*
 		 * ALTER ROLE <role> NOLOGIN NOINHERIT..
-		 * ALTER ROLE <role> SET search_path TO ...
```

Comment:
Cool. So we still disallow `ALTER ROLE <role> NOLOGIN` that can break services but we allow setting GUC parameters - which is safe.

Reply:
Yup - we still disallow `alter role` & `alter role set` for reserved roles